services/database/migrations/20251009141621_add_external_verification.sql

```diff
@@ -0,0 +1,7 @@
+-- migrate:up
+
+ALTER TABLE verification_jobs ADD external_verification json NULL;
```

Comment:
I tend to say that we can use `jsonb` for the column, because we might do some queries on the keys and query by verifier. Or what do you think?